docs/ref/contrib/admin/admindocs.txt

```diff
@@ -108,6 +108,20 @@ For example::
         context = {'mymodel': MyModel.objects.get(slug=slug)}
         return render(request, 'myapp/my_template.html', context)
 
+Due to the fact ``__qualname__`` was only introduced in Python 3, Django cannot
```

Comment:
Given that Python 2 is near EOL and the fact at I don't think many users are writing their own views this way, I don't think we need this documentation.